tower-grpc-interop/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+
+  server-go:
+    build:
+      context: .
+      dockerfile: Dockerfile-server-go
```

Comment:
do we really need to rebuild this docker image on every build? can't we just publish it? what does it need from the local repo?

Reply:
i wasn't sure whether a docker image of grpc-go's interop test server was something we would _want_ to publish or not? if it is, i can certainly try and make that change.